src/unit.rs

```diff
@@ -342,8 +342,6 @@ fn connect_socket(unit: &Unit, hostname: &str, use_pooled: bool) -> Result<(Stre
 /// Send request line + headers (all up until the body).
 #[allow(clippy::write_with_newline)]
 fn send_prelude(unit: &Unit, stream: &mut Stream, redir: bool) -> io::Result<()> {
-    //
-
```

Comment:
unrelated change?

Reply:
Just cleaning up (what seemed like) noise. If this is intentional I'll restore it.

Reply:
it's my quirk. no biggie ;)